controllers/app.controllers.js

```diff
@@ -69,6 +61,15 @@ const postCommentByArticleId = (request, response, next) => {
     .catch(next);
 };
 
+const deleteComment = (request, response, next) => {
+  const { commnent_id } = request.params;
+  removeComment(commnent_id)
```

Comment:
I'm not 100% this should be working, it looks like there is a typo on `commnent_id` that should be `comment_id`
but in _app.js_ on the path it's spelt correctly so I would have thought this would give a value of `undefined`